src/renderer/notification-handler.ts

```diff
@@ -62,14 +65,15 @@ export default class NotificationHandler {
      */
     public setupNotificationPosition() {
         // This feature only applies to windows
-        if (isMac) {
+        if (!isMac || !app.isReady()) {
```

Comment:
on `windows`, this condition will always be true and it'll be returned. Is this correct logic?

Reply:
fixed 😁